README.md

```diff
@@ -27,6 +27,39 @@ domain_props.ingress_domain  # returns myapp-ingress.myapp.acme.org
 
 </details>
 
+<details>
+  <summary>CrossAccountDomainDelegation</summary>
+
+Creates delegated domains
+
+```python
+from aws_cdk import App, Stack, Environment
+from ca_cdk_constructs import CrossAccountDomainDelegation, R53ParentZoneConfig
+
+app = App()
+
+subdomain_stack = Stack(app, "SubDomainStack", env=Environment(account="444444444444"))
+
+zone = PublicHostedZone(subdomain_stack, "Zone", zone_name="my-subdomain.acme.org")
+
+# creates delegation records in for my-subdomain.acme.org in acme.org
+CrossAccountDomainDelegation(
+    subdomain_stack,
+    "MySubdomainDnsDelegation",
+    parent_zone_config=R53ParentZoneConfig(
+        account_id="1234566789012",
+        zone_name="acme.org",
+        role_name="R53UpdateRole" # existing role in the parent zone account
```

Comment:
isn't there a way to create the role in the CDK instead of requiring external setup?

Reply:
there is but this would require `iam:CreateRole` permission to create the role in the target account. This can be dangerous and would also require cdk bootstrapping the target account with the appropriate `--cloudformation-execution-policies`